src/Aquifer.Data/AquiferDbContext.cs

```diff
@@ -8,27 +8,38 @@ namespace Aquifer.Data;
 
 public class AquiferDbContext : DbContext
 {
-    private readonly DbContextOptions<AquiferDbContext> _options;
```

Comment:
`AquiferDbReadOnlyContext` uses a `DbContextOptions<AquiferDbReadOnlyContext>` which couldn't be passed down to its base class constructor in `AquiferDbContext` so I had to make it use the common `DbContextOptions` base class.  It's only used for new-ing up an `AquiferDbContext` in the `ResourceStatusChangeHandler`.